core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -114,6 +114,7 @@ public static UnderFileSystem create(String path, Object ufsConf) {
         return factory.create(path, ufsConf);
       } catch (Exception e) {
         errors.add(e);
+        LOG.warn("Failed to create ufs", e);
```

Comment:
We don't log the exceptions at the end, we throw an exception which doesn't container stack trace information